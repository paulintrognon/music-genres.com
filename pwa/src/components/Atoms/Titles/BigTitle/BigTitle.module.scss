.container {
  font-family: Montserrat;
  text-transform: uppercase;
  font-size: 2.25rem;
}
