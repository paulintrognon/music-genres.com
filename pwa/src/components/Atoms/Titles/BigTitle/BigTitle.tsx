import classnames from 'classnames'
import styles from './BigTitle.module.scss'

type Props = React.ComponentProps<'span'>
const BigTitle: React.FC<Props> = ({ className, children }) => {
  return <span className={classnames(styles.container, className)}>{children}</span>
}
export default BigTitle
