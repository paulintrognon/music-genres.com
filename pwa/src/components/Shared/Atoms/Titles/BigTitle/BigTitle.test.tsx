import BigTitle from './BigTitle'
import { shallow } from 'enzyme'

it('should render the BigTitle', () => {
  const wrapper = shallow(<BigTitle className="testClass">Big Title</BigTitle>)
  expect(wrapper).toMatchSnapshot()
})
