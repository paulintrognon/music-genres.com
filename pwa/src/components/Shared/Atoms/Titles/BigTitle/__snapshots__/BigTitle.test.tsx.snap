// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the BigTitle 1`] = `
<span
  className="container testClass"
>
  Big Title
</span>
`;
